src/phoenix/config.py

```diff
@@ -331,5 +353,85 @@ def get_web_base_url() -> str:
     return get_base_url()
 
 
+class LoggingMode(Enum):
+    AS_LIBRARY = "default"
+    AS_APPLICATION = "structured"
+
+
+def get_env_logging_mode() -> LoggingMode:
+    logging_mode = os.getenv(ENV_LOGGING_MODE) or LoggingMode.AS_LIBRARY.value
+    if (logging_mode_lower := logging_mode.lower()) in LoggingMode._value2member_map_:
+        return LoggingMode(logging_mode_lower)
+    raise ValueError(
+        f"Invalid value `{logging_mode}` for env var `{ENV_LOGGING_MODE}`. "
+        f"Valid values are: {log_a_list([mode.value for mode in LoggingMode],'and')} "
+        "(case-insensitive)."
+    )
```

Comment:
```suggestion
    try:
        return LoggingMode(os.getenv(ENV_LOGGING_MODE, LoggingMode.AS_LIBRARY.value))
    catch ValueError:
        raise ValueError(
            f"Invalid value `{logging_mode}` for env var `{ENV_LOGGING_MODE}`. "
            f"Valid values are: {log_a_list([mode.value for mode in LoggingMode],'and')} "
            "(case-insensitive)."
        )
```

We can avoid using `_value2member_map_`, which is not part of the public interface.

Reply:
Thanks, done